vncviewer/GestureHandler.cxx

```diff
@@ -180,6 +171,15 @@ bool GestureHandler::hasState(unsigned char state) {
   return state && !(state & (state - 1));
 }
 
+int GestureHandler::isTracked(XIDeviceEvent ev) {
```

Comment:
Shouldn't this be `bool` given the usage?

Reply:
I'd like it to return an index to the relevant event in the `tracked` vector if it exists, `-1` if it doesn't.

Reply:
The caller of `isTracker()` doesn't seem to care about such details. :)

If you want to keep that return value, then I would suggest a different naming. "is" heavily implies a boolean return value.